src/dialects/postgres/schema/columncompiler.js

```diff
@@ -63,11 +63,15 @@ assign(ColumnCompiler_PG.prototype, {
   },
   smallint: 'smallint',
   tinyint: 'smallint',
-  datetime(without) {
-    return without ? 'timestamp' : 'timestamptz';
+  datetime(useTz = true, precision) {
```

Comment:
I agree. Even though the reversed logic ("don't use timezone") is confusing, it's better than breaking compatibility. Not storing time zones with time stamps can easily go unnoticed if all developers and test systems are in the same time zone, but in production it can result in data loss!